speaking-portal-project/src/index.ts

```diff
@@ -22,10 +24,14 @@ async function main () {
         console.log("Printing phoneme timings from json file...")
         console.log(phonemeContents)
 
+        console.log("Converting timings to input file...")
+        console.log(mouthCuesToInputFile({ mouthCues: phonemeContents }))
+        console.log("Generating output video...")
+        await ffmpegProcessor('../rhubarb/en-Amber.wav', 'input.txt')
+
     } catch (err: any) {
```

Comment:
For future we want to avoid using 'any', but for now it should be okay

Reply:
the `err: any` was me on the exception handling branch, and I'm looking to improve it this week anyway :)